### DISCUSSION TITLE
Predictions not getting better

### DISCUSSION THREAD

**Discussion by mj927** [talmolab/sleap#1872] [Help!] [answered]
Hi,

I have a trained model, and I added new videos to the model, but the predictions on the new video were not as good as the predictions on the videos that were previously in the model. Do I need to label the new videos too?

Thanks!

**Comment by talmo** [comment, marked as answer]
Hi @mj927,

Yep, the model you train will work well on data that are similar to those that you labeled. In order to achieve better generalization, you'll want to annotate frames across a good number (15-30) and diversity of videos (different days, animals, setups).

You won't need to do this for _every_ new video once you have sufficient representation of the diversity of your data within your labels.

Tip: don't overlabel a single video! It's much better to label 100 frames across 10 videos than to label 100 frames in the same video.

Let us know if you have any questions. It'll also be helpful if you can provide screenshots of examples that work well and don't, as well as of your labeling project as a whole so we know what you're working with in terms of animal, skeleton, number of videos, and so on.

Cheers,

Talmo

**Reply by mj927** [reply]
Thank you so much! Here are examples of a prediction that worked and one that didn't:
![Screenshot 2024-07-23 125612](https://github.com/user-attachments/assets/42fbe56c-3e5a-4117-ada5-87c6dab0dde7)
![Screenshot 2024-07-23 125703](https://github.com/user-attachments/assets/f7cf7873-b312-49b4-9454-8ff6e8015e00)

**Reply by talmo** [reply]
Ah interesting, that looks like it should do well on the second frame.

Are you using a bottom-up model?

Have you tried setting rotation augmentation to +- 180 (default is 15)?

If you don't mind sharing a screenshot of the model configuration(s) that would help too. Thanks!

**Reply by mj927** [reply]
This is the model configuration:
![Screenshot 2024-07-23 152829](https://github.com/user-attachments/assets/82f2acd7-b004-4e97-a4e0-075b155d50f6)
